src/Codeception/Step/TryTo.php

```diff
@@ -1,18 +1,24 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Codeception\Step;
 
 use Codeception\Lib\ModuleContainer;
 use Codeception\Util\Template;
+use Exception;
+use function codecept_debug;
+use function strpos;
+use function ucfirst;
 
 class TryTo extends Assertion implements GeneratedStep
 {
-    public function run(ModuleContainer $container = null)
+    public function run(ModuleContainer $container = null): bool
     {
         $this->isTry = true;
         try {
             parent::run($container);
-        } catch (\Exception $e) {
+        } catch (Exception $e) {
             codecept_debug("Failed to perform: {$e->getMessage()}, skipping...");
             return false;
         }
```

Comment:
`getTemplate` method below returns either `Template` or null.
Please check where `getTemplate` is used and make it more consistent.